frontend/components/Footer/index.js

```diff
@@ -110,87 +110,89 @@ const footerLinks: Array<Array<Link>> = [
     ],
 ];
 
-const Footer = styled('footer')({
-    backgroundColor: '#333',
-    color: '#dcdcdc',
-    fontFamily: textSans,
-    fontSize: '14px',
-
-    a: {
-        color: '#dcdcdc',
-        textDecoration: 'none',
-    },
-
-    '.footer-links': {
-        display: 'flex',
-        flexWrap: 'wrap',
-        flexDirection: 'row',
-        borderTop: '1px solid #434343',
-
-        [tablet]: {
-            borderTop: 'none',
-        },
-
-        a: {
-            paddingBottom: '12px',
-            display: 'block',
-        },
-
-        ul: {
-            width: '50%',
-            borderLeft: '1px solid #434343',
-
-            [until.tablet]: {
-                ':nth-child(odd)': {
-                    borderLeft: 0,
-                    paddingLeft: 0,
-                },
-
-                ':nth-child(3)': {
-                    paddingTop: 0,
-                },
-                ':nth-child(4)': {
-                    paddingTop: 0,
-                },
-            },
-
-            [until.leftCol]: {
-                ':nth-child(1)': {
-                    borderLeft: 0,
-                    paddingLeft: 0,
-                },
-            },
-
-            [tablet]: {
-                margin: '0 10px 36px 0',
-                flex: '1 0 0',
-            },
-
-            padding: '12px 0 0 10px',
-        },
-    },
-
-    '.email-sub__iframe': {
-        paddingTop: '12px',
-
-        [leftCol]: {
-            float: 'left',
-            width: '300px',
-            marginRight: '180px',
-        },
-    },
-
-    '.footer-inner': {
-        paddingBottom: '6px',
-    },
-
-    '.copyright': {
-        fontSize: '12px',
-        padding: '6px 0 18px',
-        borderTop: '1px solid #434343',
-        marginTop: '12px',
-    },
-});
+const footer = css`
+    background-color: #333;
+    color: #dcdcdc;
+    font-family: ${textSans};
+    font-size: 14px;
+`;
+
+const footerInner = css`
+    padding-bottom: 6px;
+`;
+
+const emailSignup = css`
+    padding-top: 12px;
+
+    ${leftCol} {
+        float: left;
+        width: 300px;
+        margin-right: 180px;
+    }
+`;
+
+const footerLink = css`
+    color: #dcdcdc;
+    text-decoration: none;
+    padding-bottom: 12px;
+    display: block;
+
+    :hover {
+        text-decoration: underline;
+    }
+`;
+
+const footerList = css`
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    border-top: 1px solid #434343;
+
+    ${tablet} {
+        border-top: none;
+    }
+
+    ul {
```

Comment:
Nesting 😬 but I think this is probably okay here as it's only one level and avoids having an extra, even more specifically named, css rule for the list in the div.

Reply:
I think it's useful because it keeps the styles for the flex parent and flex children together.